hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/server/api/protocolrecords/FederationQueueWeight.java

```diff
@@ -166,4 +179,32 @@ public static void checkHeadRoomAlphaValid(String headRoomAlpha) throws YarnExce
   protected static boolean isNumeric(String value) {
     return NumberUtils.isCreatable(value);
   }
+
+  @Public
+  @Unstable
+  public abstract String getQueue();
+
+  @Public
+  @Unstable
+  public abstract void setQueue(String queue);
+
+  @Public
+  @Unstable
+  public abstract String getPolicyManagerClassName();
+
+  @Public
+  @Unstable
+  public abstract void setPolicyManagerClassName(String policyManagerClassName);
+
+  @Override
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    builder.append("FederationQueueWeight{");
+    builder.append("Queue:" + getQueue() + ", ");
```

Comment:
Use append()

Reply:
Thank you very much for reviewing the code! I will fix it.